<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1591</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YAYA - YOLO annoter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Image scaling:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="imageScalingComboBox">
            <item>
             <property name="text">
              <string>Resize</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ResizeAspectRatio</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OriginalSize</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="gallery"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="imagePreview">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="imageCorelation"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutRight">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="fileLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Filename (number/all)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="sliderLayout">
          <item>
           <widget class="QLabel" name="fileNumberSliderLabel">
            <property name="text">
             <string>Slider label</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="prevFileButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons/tango.qrc">
              <normaloff>:/icons/16x16/media-skip-backward.png</normaloff>:/icons/16x16/media-skip-backward.png</iconset>
            </property>
            <property name="shortcut">
             <string>,</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="nextFileButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons/tango.qrc">
              <normaloff>:/icons/16x16/media-skip-forward.png</normaloff>:/icons/16x16/media-skip-forward.png</iconset>
            </property>
            <property name="shortcut">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SaveFileAnnotationsButton">
            <property name="text">
             <string>(S)ave</string>
            </property>
            <property name="icon">
             <iconset resource="icons/tango.qrc">
              <normaloff>:/icons/16x16/document-save-as.png</normaloff>:/icons/16x16/document-save-as.png</iconset>
            </property>
            <property name="shortcut">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteImageAnnotationsButton">
            <property name="text">
             <string>(X)Delete</string>
            </property>
            <property name="icon">
             <iconset resource="icons/tango.qrc">
              <normaloff>:/icons/16x16/process-stop.png</normaloff>:/icons/16x16/process-stop.png</iconset>
            </property>
            <property name="shortcut">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="fileSummaryLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="fileSelectorTableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Annotations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="detectAnnotationsButton">
            <property name="text">
             <string>(D)etect </string>
            </property>
            <property name="icon">
             <iconset resource="icons/tango.qrc">
              <normaloff>:/icons/16x16/camera-photo.png</normaloff>:/icons/16x16/camera-photo.png</iconset>
            </property>
            <property name="shortcut">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1591</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOtworzLokacje"/>
    <addaction name="actionZapisz"/>
    <addaction name="actionZamknijProgram"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSave_screenshoot"/>
    <addaction name="actionSave_copy"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOtw_rz">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionOtworzLokacje">
   <property name="text">
    <string>Open directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionZamknijProgram">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string comment="S">Ctrl+S</string>
   </property>
  </action>
  <action name="actionNextLocation">
   <property name="text">
    <string>Następna lokacja</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrevLocation">
   <property name="text">
    <string>Poprzednia lokacja</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionMountRO">
   <property name="text">
    <string>Przemontuj lokacje</string>
   </property>
  </action>
  <action name="actionNextConfiguration">
   <property name="text">
    <string>Następna konfiguracja</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionPrevConfiguration">
   <property name="text">
    <string>Poprzednia konfiguracja</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionSave_screenshoot">
   <property name="text">
    <string>Save screenshoot</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionSave_copy">
   <property name="text">
    <string>Save copy</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/tango.qrc"/>
 </resources>
 <connections/>
</ui>
